CMakeLists.txt

```diff
@@ -23,17 +23,17 @@ project(Irrlicht LANGUAGES CXX C)
 # CUDA test
 include(CheckLanguage)
 check_language(CUDA)
-enable_language(CUDA)
```

Comment:
what error did you get withthis?

Reply:
https://cmake.org/cmake/help/v3.2/module/CheckLanguage.html

hmm seems I should check for `CMAKE_CUDA_COMPILER` before proceeding to enable language

Reply:
>what error did you get withthis?

"no CUDA toolset found"

Reply:
ok i fixed it